rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -1146,7 +1135,7 @@ def _sample_episodes(
         num_env_steps_sampled = batch_size_B
         num_episodes_per_sample = len(sampled_episode_idxs)
         num_env_steps_per_sample = len(sampled_env_step_idxs)
-        sampled_n_step = (sum(sampled_n_steps) / batch_size_B,)
+        sampled_n_step = sum(sampled_n_steps) / batch_size_B
```

Comment:
Not sure why this was here before but the previous implementation of stats quietly accepted it.

Reply:
This happens if we put a comma after a line and then the linter runs over it and thinks we wanna have a tuple :)